### PR TITLE
fix: changes in the layout of the main page on github

### PR DESCRIPTION
**Pull request by abarmina**
added missing elements according to the current version of the page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,10 +50,10 @@
     "build-pages": "concurrently \"npm run build-pages:ts\" \"npm run build-pages:less\" && npm run build-pages:eleventy",
     "build-pages:ts": "cd pages && webpack",
     "build-pages:less": "lessc pages/src/localdev.less pages/static/bundles/localdev.css --source-map=pages/static/bundles/localdev.css.map",
-    "build-pages:eleventy": "eleventy",
+    "build-pages:eleventy": "ELEVENTY_ENV=production npx @11ty/eleventy",
```
Comment (ala-n) [inline package.json:53]: Is it working in all OS?
It seems like I have a problem with running it in such way on WIndows
Mb it has an option to pass environment as a regular command-line argument?

Comment (ala-n) [inline package.json:53]: Is it working in all OS?
It seems like I have a problem with running it in such a way on WIndows
Mb it has an option to pass environment as a regular command-line argument?

Comment (abarmina) [reply package.json:53]: you're right, it doesn't work on Windows. I rewrote this using a command-line argument

**Review by ala-n — APPROVED** [review]

File: .eleventy.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
 module.exports = config => {
+  config.setUseGitIgnore(false);
+  config.addWatchTarget("pages/static/bundles");
```
Comment (dshovchko) [inline .eleventy.js:3]: typo: double quotes

File: pages/views-11ty/data/env.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  environment: process.argv.find(arg => arg.startsWith("--env="))?.split("=")[1],
```
Comment (dshovchko) [inline pages/views-11ty/data/env.js:2]: typo: double quotes

**Review by dshovchko — APPROVED** [review]

File: .eleventy.js (hunk 2 above)
Comment (abarmina) [reply .eleventy.js:3]: oops, thank you

File: pages/views-11ty/data/env.js (hunk 3 above)
Comment (abarmina) [reply pages/views-11ty/data/env.js]: thanks, I fixed it

**Review by YanaBr — APPROVED** [review]

**Review by andreybelous — APPROVED** [review]

**Pull request merged**